### DISCUSSION TITLE
PrimereactDatatable: Get number of currently displayed rows

### DISCUSSION THREAD

**Discussion by Amit-ga** [primefaces/community#1996] [PrimeReact]
Hello,

My table toggles between Virtual Scroll - lazy state and regular paginator state according the number of displayed rows which I control via my rows variable and the table 'rows' attribute.
I noticed the rows variable does not get updated when filter is applied, meaning: the number of rows **to display** stays the same (because it really has not change upon applying the filter), but the number of **displayed rows** can be lower, and I dont know how to get access to it at a given point in time, is that possible?

Thanks!

**Comment by Amit-ga** [comment]
Hi all! could really use this. please let me know if itspossible.
When i filter or preform some kind of operation on the table, i want to know how many rows are currently displayed.
Lets say I have 500 rows in total, and rows per page is set to a 100.
Now i filter Age>=25, I want to know how many rows meet that criteria.

Thank you.

**Reply by melloware** [reply]
isn't that this example: https://primereact.org/datatable/#paginator_template you can use the `CurrentPageReport` in the Paginator template and it shows you the current total rows?

**Reply by Amit-ga** [reply]
yes, it is.
what is the proper syntax to access this number using the dt ref?

Thank you
